src/libmongoc/doc/mongoc_uri_t.rst

```diff
@@ -211,7 +212,7 @@ MONGOC_URI_W                               w                                 Det
                                                                              * majority = For replica sets, if you specify the special majority value to w option, write operations will only return successfully after a majority of the configured replica set members have acknowledged the write operation.
                                                                              * n = For replica sets, if you specify a number n greater than 1, operations with this write concern return only after n members of the set have acknowledged the write. If you set n to a number that is greater than the number of available set members or members that hold data, MongoDB will wait, potentially indefinitely, for these members to become available.
                                                                              * tags = For replica sets, you can specify a tag set to require that all members of the set that have these tags configured return confirmation of the write operation.
-MONGOC_URI_WTIMEOUTMS                      wtimeoutms                        The time in milliseconds to wait for replication to succeed, as specified in the w option, before timing out. When wtimeoutMS is 0, write operations will never time out.
+MONGOC_URI_WTIMEOUTMS                      wtimeoutms                        Deprecated in favor of MONGOC_URI_TIMEOUTMS. The time in milliseconds to wait for replication to succeed, as specified in the w option, before timing out. When wtimeoutMS is 0, write operations will never time out.
```

Comment:
I think we need to deprecate `mongoc_write_concern_set_wtimeout` and `mongoc_write_concern_get_wtimeout` as well.

Edit: I think this will be done in CDRIVER-3842. No change needed here!

Reply:
Acknowledged.